install-guide.md

```diff
@@ -51,20 +53,93 @@ sudo pacman -S --needed libunwind glfw-x11 vulkan-devel sox git cmake alsa-lib n
 go to [Continued Install.](/wiki/installation/#continued-install)
 
 #### Continued Install
-2. Clone the repo.
+
+1. Clone the repo.
+
+    ```sh
+    git clone --recursive https://github.com/RPCSX/rpcsx && cd rpcsx
+    git submodule update --init --recursive
+    ```
+
+2. Compile the emulator.
+
+    ```sh
+    cmake -B build && cmake --build build -j$(nproc)
+    ```
+
+### Method 2: WSL
+
+**Install WSL**
+Run Windows Powershell as Administrator:
 
 ```sh
-git clone --recursive https://github.com/RPCSX/rpcsx && cd rpcsx
-git submodule update --init --recursive
+wsl --install
 ```
 
-3. Compile the emulator.
+Reboot your machine
+
+**Install Ubuntu:**
+Run Windows Powershell
 
 ```sh
-cmake -B build && cmake --build build -j$(nproc)
+wsl --install
 ```
 
-### Method 2: WSL
-**TODO (Coming Soon)**
+Setup user credentials
+
+**Configure Ubuntu and build RPCSX:**
+
+```sh
+sudo add-apt-repository ppa:oibaf/graphics-drivers
+sudo apt upgrade
+```
+
+```sh
+sudo apt install -y build-essential cmake libunwind-dev libglfw3-dev libvulkan-dev libsox-dev git libasound2-dev nasm g++-14
+sudo apt install -y pkgconf libasound2-plugins vainfo mesa-va-drivers
```

Comment:
Why can't this be added to the command above?

Reply:
I copy and posted DH's message from discord. I was wondering about that format as well. Glad to know I wasn't the only 1 questioning it. Do you think maybe WSL vs Standard Ubuntu install instructions?

Reply:
Ubuntu -
 sudo apt install build-essential cmake libunwind-dev libglfw3-dev libvulkan-dev vulkan-validationlayers-dev libsox-dev git libasound2-dev nasm g++-14

Yeah, DH was showcasing Standard vs WSL specific packages. I'll add commentary